File: urionlinejudge/1015 - Distance Between Two Points.cpp
// Created by islam elhady on 9/5/20.

#include <iostream>
#include <math.h>
 
using namespace std;
 
int main() {
 
    double x1, x2, y1, y2, dist;

    scanf("%lf %lf %lf %lf", &x1, &y1, &x2, &y2);

    dist = sqrt(pow(x2-x1,2)+pow(y2-y1,2));

    printf("%.4lf\n", dist);
    
    return 0;
}

File: urionlinejudge/1015 - Distance Between Two Points.java
// Created by islam elhady on 1/21/18.

import java.io.IOException;
import static java.lang.Math.sqrt;
import java.util.Scanner;
public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner s = new Scanner (System.in);              
             
         double x1,y1,x2,y2 ;
          x1 = s.nextDouble();
          y1 = s.nextDouble();
          x2 = s.nextDouble();
          y2 = s.nextDouble();
           x1= x2-x1;
           y2= y2-y1;
          
           x2=x1*x1;
           y1=y2*y2;
          System.out.printf( "%.4f\n",    sqrt(x2+y1) );
 
    }
 
}